### PR TITLE
This commit adds support to use Email as a channel on Verify

### PR DESCRIPTION
**Pull request by leandroshp**
The changes included in this changes are:
(1) "email" is now a supported verify type
(2) The property Recipient in the Verify object changed type from int to string to support email addresses. The Create methos signature was already expecting a string as the recipient so that was handy for this change to not be a bit breaking change to current users. If for some reasons the Recipient property was accssed directly, it might break current usage.
(3) The property Subject was added to enable SDK users to specify the Subkect of the OTP email
(4)The method GetVerifyEmailMessage was added to retrieve a email message object
(5) Tests for new method were added and previous tests were fix as change (1) broke them.

### CODE REVIEW

File: verify/verify.go [hunk 1]
```diff
@@ -98,6 +106,18 @@ func VerifyToken(c *messagebird.Client, id, token string) (*Verify, error) {
 	return verify, nil
 }
 
+func GetVerifyEmailMessage(c *messagebird.Client, id string) (*VerifyMessage, error) {
```
Comment (epels) [inline verify/verify.go:109]: I'd suggest renaming to `ReadVerifyEmailMessage` for consistency with other methods for fetching serverside resources

File: verify/verify.go [hunk 2]
```diff
@@ -18,7 +18,12 @@ type Verify struct {
 	Messages           map[string]string
 	CreatedDatetime    *time.Time
 	ValidUntilDatetime *time.Time
-	Recipient          int
+	Recipient          string
+}
+
+type VerifyMessage struct {
+	Id     string `json:"id"`
```
Comment (epels) [inline verify/verify.go:25]:
```suggestion
	ID     string `json:"id"`
```

File: verify/verify.go (hunk 1 above)
Comment (leandroshp) [reply verify/verify.go]: done

**Comment by leandroshp** [comment]
@epels I have included your suggestions and also the fix for the case in which the Recipient was still being returned as a number from the backend and causing an issue. Can you check again?

**Comment by epels** [comment]
LGTM! 👍

**Pull request merged**